base/ca/src/com/netscape/ca/CRLIssuingPoint.java

```diff
@@ -1599,6 +1619,15 @@ public long getNextUpdateGracePeriod() {
      */
     private long findNextUpdate(boolean fromLastUpdate, boolean delta) {
         long now = System.currentTimeMillis();
+
+        //If we have already created a future thisUpdate value, make "now" this time in the future.
+        long futureNow = 0;
+        if(this.mFutureThisUpdateValue!= null) {
```

Comment:
Any reason why mForbidFutureThisUpdateValue is not checked here, but instead is checked down below?

Reply:
Actually the forbid flag is checking in UpdateCRLNow method before the call to findNextUpdate where you are  referencing here...